18F/identity-idp
# Remove redundant "remaining" value from session timeout API (Part 1)

## 🛠 Summary of changes

Extracted from #7966

Removes the `remaining` value from the session active / keepalive endpoints. This is an iteration toward a refactoring of session timeout behaviors, partially implemented in #7966.

The `remaining` value is unnecessary since, just as it was computed server-side, it can be derived on the client-side by comparing the `timeout` and current time. Arguably this would produce a more accurate value, since it would not be subject to time drift delays in the same way that a network response would be.

## 📜 Testing Plan

1. Sign in
2. Wait for session timeout modal
3. Observe that redirect happens once countdown reaches 0
   - This is an effective test since the `timeRemaining` is used to schedule the final polling request if the session inactivation would occur sooner than the regularly scheduled poll 

It's easier to test by setting a very low session timeout in local `config/application.yml`:

```yml
session_timeout_in_minutes: 1
```
